viewpump/build.gradle

```diff
@@ -29,6 +29,10 @@ android {
         textOutput 'stdout'
         textReport System.getenv('CI') == 'true'
     }
+    libraryVariants.all {
+        // TODO replace with https://issuetracker.google.com/issues/72050365 once released.
+        it.generateBuildConfig.enabled = false
```

Comment:
do any other tools depend on this? (something like fabric that needs to check if debug or not)

Reply:
Nope! We do the same in autodispose as well